src/amm_loop_functions.cpp

```diff
@@ -71,7 +77,65 @@ double BarrZeeLoopFS(double z)
 }
 
 /**
- * Barr-Zee 2-loop function with scalar loop (arXiv:1502.04199 Eq 27).
+ * Barr-Zee 2-loop function with fermion loop and pseudoscalar and Z
+ * boson mediators.
+ *
+ * @param x squared mass ratio (mf/ms)^2.
+ * @param y squared mass ratio (mf/mz)^2.
+ */
+double BarrZeeLoopFPSZ(double x, double y)
+{
+   if (x < 0 || y < 0) {
+      throw OutOfBoundsError("BarrZeeLoopFPSZ: arguments must not be negative.");
+   } else if (y == 0) {
+      return 0;
+   } else if (x == y) {
+      if (x < std::numeric_limits<double>::epsilon()) {
+         const double pi26 = 1.6449340668482264; // Pi^2/6
```

Comment:
Maybe `consexpr` or `static constexpr`

Reply:
Thanks! Fixed in [e6c5c8c](https://github.com/FlexibleSUSY/FlexibleSUSY/pull/519/commits/e6c5c8ce95ca693170b83495f682302c109d5e72)